src/common/SPELLS/index.ts

```diff
@@ -91,8 +93,13 @@ const SPELLS = new Proxy(InternalSpellTable, {
 });
 
 export default SPELLS;
-export const maybeGetSpell = (key: string | number | undefined): Spell | undefined =>
-  key ? InternalSpellTable[key as any] : undefined;
+
+export function maybeGetSpell(key: string | number | undefined, expansion = Expansion.Dragonflight): Spell | undefined {
```

Comment:
note for later: we probably want an enum to represent which *game* you're looking at (Retail or Classic) instead of the expansion to avoid having to change all the calls to this later.

you don't have to worry about fixing that. i just am making a note so i remember

Reply:
We can add that to the expansion context. You can have it provide `isClassic` and `isRetail` pretty easily.